### DISCUSSION TITLE
How would I change the color of the corners when a qr code is accepted?

### DISCUSSION THREAD

**Discussion by Pavel-Sushko** [mebjas/html5-qrcode#276] [General]
So I need to scan tickets, and when it's a valid ticket I want it to be green (like it is now), but when the ticket is invalid, I want them to be red.

Thanks

**Comment by ROBERT-MCDOWELL** [comment]
so you need first to manage the data onSuccess() scanned from the qrcode to your app/server and return god when good and when bad just show an error in red....

**Reply by Pavel-Sushko** [reply]
No, I need the actual borders around the QR code area to be red, like when they turn green when detecting. Instead of turning green when they detect something, I would like to know how to make them turn any color of my own accord

**Comment by ROBERT-MCDOWELL** [comment]
I think you don't catch it. the scanner scan the QR code the number of frame you set when you create the scanner object.
so to be more clear every frame that does not succeed will be read, thus all the time and green when the qr code is successful
